### PR TITLE
Added DynamoAddinGenerator.prj to DynamoRevit.2013.sln

### PR DESCRIPTION
**Pull request by sh4nnongoh**
### Purpose

To create a DynamoRevit MSI that is independent on DynamoCore files.

* A new project called DynamoAddInGenerator was created in the tools directory.
This project was referenced from a similar project in Dynamo, but has been edited heavily to suit the new requirements.
This project uses DynamoInstallDetective.dll & RevitAddIn.dll to generate an executable that is used to generate/delete Dynamo Revit Addins.
The referenced dlls are stored in the tools/install/Extra directory.

This project is dependent on the configuration changes made in:
https://github.com/DynamoDS/DynamoRevit/pull/852

YouTrack: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9574

### Reviewers

@sharadkjaiswal

### FYIs

@Benglin

### CODE REVIEW

File: src/Tools/DynamoAddInGenerator/DynamoAddInGenerator.csproj [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <Optimize>true</Optimize>
+    <OutputPath>$(OutputPath)\..</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="DynamoInstallDetective">
+      <HintPath>..\..\..\tools\install\Extra\DynamoInstallDetective.dll</HintPath>
```
Comment (sharadkjaiswal) [inline src/Tools/DynamoAddInGenerator/DynamoAddInGenerator.csproj]: Let's not check-in DynamoInstallDetective.dll in this repo. You can reference it using $(DYNAMOAPI)\DynamoInstallDetective.dll

File: src/Tools/DynamoAddInGenerator/DynamoAddInGenerator.csproj [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  </ImportGroup>
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{92A46535-D870-4E1A-AED0-7492789E9C4A}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>DynamoAddInGenerator</RootNamespace>
+    <AssemblyName>DynamoAddInGenerator</AssemblyName>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```
Comment (sharadkjaiswal) [inline src/Tools/DynamoAddInGenerator/DynamoAddInGenerator.csproj]: Let this guy inherit TargetFrameworkVersion from CS.props

File: src/Tools/DynamoAddInGenerator/DynamoAddInGenerator.csproj [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="DynamoInstallDetective">
+      <HintPath>..\..\..\tools\install\Extra\DynamoInstallDetective.dll</HintPath>
+    </Reference>
+    <Reference Include="RevitAddinUtility">
+      <HintPath>..\..\..\tools\install\Extra\RevitAddinUtility.dll</HintPath>
```
Comment (sharadkjaiswal) [inline src/Tools/DynamoAddInGenerator/DynamoAddInGenerator.csproj]: You can check-in RevitAddinUtility to the extern folder, rather than tools folder.

**Comment by sharadkjaiswal** [comment]
This looks good, do we have similar PR for other branches?

**Comment by sh4nnongoh** [comment]
@sharadkjaiswal I have submitted the PR for the other versions

**Pull request merged**